examples/tech_meetings/20241213_run_damages_losses_without_files.ipynb

```diff
@@ -0,0 +1,274 @@
+    "Related to https://github.com/Deltares/ra2ce/issues/654."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Imports\n",
+    "\n",
+    "from pathlib import Path\n",
+    "from ra2ce.ra2ce_handler import Ra2ceHandler\n",
+    "from ra2ce.network.network_config_data.enums.aggregate_wl_enum import AggregateWlEnum\n",
+    "from ra2ce.network.network_config_data.enums.source_enum import SourceEnum\n",
+    "from ra2ce.analysis.analysis_config_data.analysis_config_data import (\n",
+    "    AnalysisConfigData,\n",
+    "    AnalysisSectionDamages,\n",
+    "    AnalysisSectionLosses,\n",
+    ")\n",
+    "from ra2ce.analysis.analysis_config_data.enums.analysis_damages_enum import (\n",
+    "    AnalysisDamagesEnum,\n",
+    ")\n",
+    "from ra2ce.analysis.analysis_config_data.enums.analysis_losses_enum import (\n",
+    "    AnalysisLossesEnum,\n",
+    ")\n",
+    "from ra2ce.analysis.analysis_config_data.enums.damage_curve_enum import DamageCurveEnum\n",
+    "from ra2ce.analysis.analysis_config_data.enums.event_type_enum import EventTypeEnum\n",
+    "from ra2ce.analysis.analysis_config_data.enums.traffic_period_enum import (\n",
+    "    TrafficPeriodEnum,\n",
+    ")\n",
+    "from ra2ce.analysis.analysis_config_data.enums.trip_purpose_enum import TripPurposeEnum\n",
+    "from ra2ce.analysis.analysis_config_data.enums.weighing_enum import WeighingEnum\n",
+    "from ra2ce.network.network_config_data.enums.aggregate_wl_enum import AggregateWlEnum\n",
+    "from ra2ce.network.network_config_data.network_config_data import (\n",
+    "    HazardSection,\n",
+    "    NetworkConfigData,\n",
+    "    NetworkSection,\n",
+    ")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "root_dir = Path.cwd().parent.joinpath(\"data\", \"adaptation\")\n",
+    "\n",
+    "static_path = root_dir.joinpath(\"static\")\n",
+    "hazard_path =static_path.joinpath(\"hazard\")\n",
+    "network_path = static_path.joinpath(\"network\")\n",
+    "output_path=root_dir.joinpath(\"output\")\n",
+    "\n",
+    "input_path = root_dir.joinpath(\"input\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Configure the network"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "_network_section = NetworkSection(\n",
+    "    source= SourceEnum.SHAPEFILE,       \n",
+    "    primary_file = [network_path.joinpath(\"network.shp\")], \n",
+    "    file_id = \"ID\",\n",
+    "    link_type_column=\"highway\",\n",
+    "    save_gpkg=True\n",
+    ")\n",
+    "\n",
+    "_hazard = HazardSection(\n",
+    "    hazard_map=[Path(file) for file in hazard_path.glob(\"*.tif\")],\n",
```

Comment:
As far as I know you do not need to cast them into `Path` as they are already a subpath from `hazard_path`. https://docs.python.org/3/library/pathlib.html#basic-use